Mention the className prop decision in README ... (#61)
airbnb/react-outside-click-handler: README.md

### display: `PropTypes.oneOf(['block', 'flex', 'inline-block', 'inline', 'contents'])`

By default, the `OutsideClickHandler` renders a `display: block` `<div />` to wrap the subtree defined by `children`. If desired, the `display` can be set to `inline-block`, `inline`, `flex`, or `contents` instead. There is no way not to render a wrapping `<div />`.

> 

Reviewer: ```suggestion
```

## FAQ

> ### I need the className or styles props, are they available?

Reviewer: ```suggestion
### I think I need the `className` or `style` props; are they available?
```

> Those particular props are not available as a design choice, [check out this article](https://medium.com/@JanPaul123/don-t-pass-css-classes-between-components-e9f7ab192785) for more in-depth reasoning. But if the props that are available don't cover your use-case, please create an issue and describe your use-case - we will be happy to accomodate it.

Reviewer: ```suggestion
Those particular props are not available as an intentional design choice. [Check out this article](https://medium.com/@JanPaul123/don-t-pass-css-classes-between-components-e9f7ab192785) for more in-depth reasoning. If the props that are available do not cover your use case, please create an issue and describe your it - we will be happy to consider accommodating it.
```
